documentation/book/assembly-kafka-broker.adoc

```diff
@@ -0,0 +1,45 @@
+// This assembly is included in the following assemblies:
+//
+// getting-started.adoc
+
+// Save the context of the assembly that is including this one.
+// This is necessary for including assemblies in assemblies.
+// See also the complementary step on the last line of this file.
+:parent-context: {ContextProduct}
+
+include::common/attributes.adoc[]
+
+[id='kafka-broker-{context}']
+= Kafka broker
+
+{ProductName} uses `StatefulSets` to deploy Kafka brokers.
```

Comment:
I think that not everyone might understand what StatefulSet is. I think we should add link here ... but to make it more complicated the link should probably go to Kubernetes documentation upstream and OCP documentation downstream. What is the right way to do that?

Reply:
We can do attributes for it to make it link different places.  I'm thinking about where to put the link though.